pkg/types/validation.go

```diff
@@ -2,9 +2,6 @@ package types
 
 import (
```

Comment:
Opinion from me - it would be better to move config and validations into strategies, this is going to get big very quickly.